plugin/inc/installsteps/handleprofilerightsinstallstep.class.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace GlpiPlugin\Iservice\InstallSteps;
+
+class HandleProfileRightsInstallStep
+{
+    const CLEANUP_ON_UNINSTALL = true;
+
+    public static function do(): bool
+    {
+        global $DB;
+
+        $table = 'glpi_profilerights';
+        $superAdminProfileId = \Profile::getSuperAdminProfilesId();
```

Comment:
Profile::getSuperAdminProfilesId() returns an array!
```suggestion
        $superAdminProfileIds = \Profile::getSuperAdminProfilesId();
```
(check if the code has to be modified later)